tools/system_libs.py

```diff
@@ -954,6 +952,7 @@ def get_libcall_files(self):
     other_files = files_in_path(
       path='system/lib/libc',
       filenames=['emscripten_memcpy.c', 'emscripten_memset.c',
+                 'emscripten_get_heap_size.c',
```

Comment:
Why did this change?

Reply:
This landed as part of a split out change